app/controllers/scanned_resources_controller.rb

```diff
@@ -64,6 +64,20 @@ def save_and_ingest
     end
   end
 
+  def around_delete_action
+    return unless @change_set.is_a? RecordingChangeSet
+
+    # Only allow deleting empty recordings whose tracks are not in playlists.
```

Comment:
It's okay if they're not empty right? Maybe juist a small comment fix.

```suggestion
    # Only allow deleting recordings whose tracks are not in playlists.
```